keyboards/ungodly/launch_pad/rules.mk

```
@@ -0,0 +1,26 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+BOOTMAGIC_ENABLE = no
+MOUSEKEY_ENABLE = no
+EXTRAKEY_ENABLE = yes
+CONSOLE_ENABLE = no
+COMMAND_ENABLE = no
+NKRO_ENABLE = yes
+BACKLIGHT_ENABLE = no
+RGBLIGHT_ENABLE = no
+MIDI_ENABLE = yes
+UNICODE_ENABLE = no
+BLUETOOTH_ENABLE = no
+AUDIO_ENABLE = no
+FAUXCLICKY_ENABLE = no
+OLED_DRIVER_ENABLE = yes
+ENCODER_ENABLE = yes
+RGB_MATRIX_ENABLE = WS2812
+
+SRC += analog.c
+EXTRAFLAGS += -flto
```

This includes some other settings that may drop the file size a bit.
```suggestion
LTO_ENABLE = yes
```